tests/conftest.py

```diff
@@ -2,6 +2,49 @@
 # See LICENSE file for licensing details.
 
 import argparse
+import os
+from unittest.mock import PropertyMock
+
+import pytest
+from ops import JujuVersion
+from pytest_mock import MockerFixture
+
+
+@pytest.fixture(autouse=True)
+def juju_has_secrets(mocker: MockerFixture):
```

Comment:
nit: consider renaming this fixture to something like `juju_version_above_3`

Reply:
Let's leave it for now -- as the same name exists on a few other charms too :-)
Furthermore it's consistent with the `JujuVersion.has_secrets` terminology, which after all is the target.

And yes, your are totally right... Good news should be that this whole fixture thing is to be dropped with Juju2 ;-) 
So it's kind of a temporary thing

Reply:
[Not to mention: `autouse` fixture, never to be referred to by name ;-) ]